src/services/apes/apes.ts

```diff
@@ -23,26 +22,35 @@ export const apes = createApi({
     tokens: builder.query<Token[], ChainID>({
       query: (chainID) => `v1/tokens/${chainID}`,
     }),
-
-    stripeSessionURL: builder.mutation<{ url: string }, StripeSessionURLParams>(
-      {
-        query: ({ endowId, liquidSplitPct }) => ({
-          url: `${v(1)}/fiat/stripe-proxy/apes/${network}`,
-          method: "POST",
-          body: JSON.stringify({
-            endowmentId: endowId,
-            splitLiq: liquidSplitPct,
-            redirectUrl: `${window.location.origin}${appRoutes.donate_fiat_thanks}`,
-          }),
+    getStripePaymentStatus: builder.query<
+      Pick<PaymentIntent, "status">,
+      { paymentIntentId: string }
+    >({
+      query: ({ paymentIntentId }) => ({
+        url: `v2/fiat/stripe-proxy/apes/staging?paymentIntentId=${paymentIntentId}`,
+      }),
+    }),
+    createStripePaymentIntent: builder.mutation<
+      { clientSecret: string },
+      StripePaymentIntentParams
+    >({
+      query: ({ amountInCents, endowmentId, liquidSplitPct }) => ({
+        url: `v2/fiat/stripe-proxy/apes/staging`,
```

Comment:
```suggestion
        url: `v2/fiat/stripe-proxy/apes/{staging}`,
```
also add `production`